podcasts/SwiftUI/ScalingValue.swift

```diff
@@ -0,0 +1,156 @@
+import SwiftUI
+
+
+/// Allows for easy scaling of the given value based on the size of the view its contained in (usually the full height)
+/// This is useful when you need to position and/or size a view consistently across device sizes
+///
+///
+/// Basic usage is: @ScalingValue(with: .width) var scaledSize = 0.5
+/// You can also specify a custom arithmetic operator (+, -, /, *) *to use in the calculations, the default is *
+///
+/// See the ExampleView in the PreviewProvider below for usage
+@propertyWrapper public struct ScalingValue<Value>: DynamicProperty where Value: BinaryFloatingPoint {
```

Comment:
I wonder if `ProportionalValue` might be a better name? Because it's always proportional to the screen size, right? Although it is quite similar to `ScalingValue` so not sure.